scripts/player/cmd/disttocore.lua

```diff
@@ -0,0 +1,10 @@
+if onServer() then
+
+function initialize()
+	player = Player()
+	local distanceFromCenter = length(vec2(Sector():getCoordinates()))
+	player:sendChatMessage("Server", 0, "Distance from core : "..math.ceil(distanceFromCenter*10)*.1)

```

Comment:
`"Distance from \\s(0:0): "…`

A community could refer to the area encompassed by the *barrier* as the *core*, so using `(0:0)` as reference removes any confusion. (The markup `\\s(…:…)` will turn the coordinates into a link that, if clicked on it, opens the galaxy map.)